topics/performance-on-cpu.md

```diff
@@ -37,23 +37,25 @@ For a proper On-CPU analysis, Valkey (and any dynamically loaded library like
 Valkey Modules) requires stack traces to be available to tracers, which you may
 need to fix first. 
 
-By default, Valkey is compiled with the `-O2` switch (which we intent to keep
-during profiling). This means that compiler optimizations are enabled. Many
-compilers omit the frame pointer as a runtime optimization (saving a register),
-thus breaking frame pointer-based stack walking. This makes the Valkey
-executable faster, but at the same time it makes Valkey (like any other program)
-harder to trace, potentially wrongfully pinpointing on-CPU time to the last
-available frame pointer of a call stack that can get a lot deeper (but
-impossible to trace).
+By default, Valkey is compiled with the `-O3` optimization flag, which enables 
```

Comment:
I think we should still have the information `(which we intend to keep during profiling).`